### PR TITLE
FT: Update buffer api

### PR DESCRIPTION
**Pull request by alexandre-merle**
Update buffer api

### CODE REVIEW

**Comment by alexandre-merle** [comment]
@ironman-machine let's go
SCALITY_INTEGRATION_BRANCH=fix/nimrod-build-dep
SCALITY_FEDERATION_BRANCH=ft/dep-patcher-ansible

File: tests/unit/auth/v4/signingKey.js [hunk 1]
```diff
@@ -11,10 +11,10 @@ describe('v4 signing key calculation', () => {
         const secretKey = 'verySecretKey1';
         const region = 'us-east-1';
         const scopeDate = '20160209';
-        const expectedOutput = '5c19c3be2935c2aa4fc296754904c2' +
-            '8b6dc2aac285635fc2b47bc3a2c293c28b08c299177e5906c394c2b17221';
+        const expectedOutput = '5c19fe2935aa4f967549048b6daa85635fb47' +
+            'be2938b0899177e5906d4b17221';
         const actualOutput = calculateSigningKey(secretKey, region, scopeDate);
-        const buff = new Buffer(actualOutput).toString('hex');
+        const buff = Buffer.from(actualOutput, 'binary').toString('hex');
```
Comment (DavidPineauScality) [inline tests/unit/auth/v4/signingKey.js]: Why use binary here, when we use "utf8" in the other buffers ?
Hence the question: why does the output change ? Is the goal not to stay compatible ?

Comment (alexandre-merle) [inline tests/unit/auth/v4/signingKey.js]: it is, but the calculatesigningkey return a binary string :)

**Comment by rahulreddy** [comment]
@ironman-machine again
SCALITY_INTEGRATION_BRANCH=fix/nimrod-build-dep
SCALITY_FEDERATION_BRANCH=ft/dep-patcher-ansible

**Comment by alexandre-merle** [comment]
Rebased :)

**Pull request merged**